ipwb/reconstructive-banner.js

```diff
@@ -54,19 +54,25 @@ class ReconstructiveBanner extends HTMLElement {
       if (diff < 0) {
         return 'Capture from the future!';
       }
+      let unit, quotient;
       if (diff >= 31622400000) {
-        return `Captured ${Math.round(diff / 31622400000)} year(s) ago`;
+        unit = 'year';
```

Comment:
We should give semantics to these magic numbers in the future.

Reply:
These numbers are milliseconds in respected bigger units of time that are not going to change. For readability purpose we can define them as constants if you prefer that way.

Reply:
@ibnesayeed Yes, defining them as constants would be an improvement.

Reply:
I have made this change in the upstream repo. Will synchronize here later.